x-pack/plugins/upgrade_assistant/server/lib/apm/index.ts

```diff
@@ -62,7 +62,7 @@ export const isLegacyApmIndex = (
 // source: https://github.com/elastic/apm-integration-testing/blob/master/tests/server/test_upgrade.py
 export const apmReindexScript = `
   // add ecs version
-  ctx._source.ecs = ['version': '1.0.0-beta2'];
+  ctx._source.ecs = ['version': '1.1.0-dev'];
```

Comment:
What is the dev pre-release tag for?

Reply:
that matches the version of the published ecs schema 7.0.1 is aligned with.  This is something inherited from libbeat, not directly set by apm-server